components/VC/MosipVCItem/MosipVCItemDetails.tsx

```diff
@@ -167,7 +168,7 @@ export const MosipVCItemDetails: React.FC<
                       weight="semibold"
                       size="smaller"
                       color={Theme.Colors.Details}>
-                      {uin}
+                      {getMaskedText(uin)}
```

Comment:
In details, are we expecting to show masked data?

Reply:
checked with @kratitva, new ui is coming up which will involve lot of ui refactoring, so for now, lets go with the existing approach. so removing this.